### PR TITLE
[Dev] Fix new culture info

### PR DESCRIPTION
**Pull request by VictoriousRaptor**
Fix main window creating a new `CultureInfo` object every second in clock/date update code.

Tests

- [x] Clock/date in main window and preview can reflect changes of languages.

### CODE REVIEW

**Comment by onesounds** [comment]
Draft?

**Comment by VictoriousRaptor** [comment]
> Draft?

Finished now.

File: Flow.Launcher/ViewModel/MainViewModel.cs [hunk 1]
```diff
@@ -334,20 +335,21 @@ private void Esc()
...
-        public CultureInfo cultureInfo => new CultureInfo(Settings.Language);
         private async Task RegisterClockAndDateUpdateAsync()
         {
             var timer = new PeriodicTimer(TimeSpan.FromSeconds(1));
             // ReSharper disable once MethodSupportsCancellation
             while (await timer.WaitForNextTickAsync().ConfigureAwait(false))
             {
                 if (Settings.UseClock)
-                    ClockText = DateTime.Now.ToString(Settings.TimeFormat, cultureInfo);
+                    ClockText = DateTime.Now.ToString(Settings.TimeFormat, Culture);
```
Comment (taooceros) [inline Flow.Launcher/ViewModel/MainViewModel.cs:347]: I am confused why this is needed. Isn't `CultureInfo.CurrentCulture` is changed whenever language is changed?

Comment (VictoriousRaptor) [reply Flow.Launcher/ViewModel/MainViewModel.cs:347]: CultureInfo.CurrentCulture is per thread.

**Review by onesounds — APPROVED** [review]

File: Flow.Launcher/ViewModel/MainViewModel.cs (hunk 1 above)
Comment (VictoriousRaptor) [reply Flow.Launcher/ViewModel/MainViewModel.cs:347]: Updated, plz check again.

**Review by taooceros — APPROVED** [review]

**Pull request merged**